config/crd/bases/konnect.konghq.com_konnectcontrolplanes.yaml

```diff
@@ -0,0 +1,206 @@
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
```

Comment:
Ditto this.

Reply:
Good catch. 3c98f5c5e980db9a2b23a0bc031b80c49faded45 removes this.